desktop/src/main/java/bisq/desktop/main/offer/offerbook/OfferBookView.java

```diff
@@ -1169,13 +1169,15 @@ public void updateItem(final OfferBookListItem item, boolean empty) {
                                         button.setStyle(CssTheme.isDarkTheme() ? "-fx-text-fill: white" : "-fx-text-fill: #444444");
                                         button.setOnAction(e -> onRemoveOpenOffer(offer));
 
-                                        iconView2.setId("image-edit");
-                                        button2.updateText(Res.get("shared.edit"));
-                                        button2.setId(null);
-                                        button2.setStyle(CssTheme.isDarkTheme() ? "-fx-text-fill: white" : "-fx-text-fill: #444444");
-                                        button2.setOnAction(e -> onEditOpenOffer(offer));
-                                        button2.setManaged(true);
-                                        button2.setVisible(true);
+                                        if (!offer.isBsqSwapOffer()) {
```

Comment:
We probably need to do something like that:
```
if (!offer.isBsqSwapOffer()) {
    // ...
} else {
  button2.setManaged(false);
  button2.setVisible(false);
}
```
IIRC `updateItem` is re-rendering items on list during scrolling.

Reply:
Thanks for reviewing! I refactored accordingly.

Reply:
Thank you - looks good now!